### PR TITLE
Assignment_13 commit

### PR DESCRIPTION
**Pull request by naviya89**
Create a Parent class which has a String surname = “TYSON”, And create Child class having String  firstName = “MIKE”.
Create a Test Class. In that create a Child class object and call getSurname and getFirstName. 
Note - Use inheritance concept

### CODE REVIEW

File: src/main/java/com/training/assignments/Parent.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.training.assignments;
+
+public class Parent {
+    public String getSurname() {
+        String surname = "TYSON";
+        return surname;
```
Comment (shrikanttechincal) [inline src/main/java/com/training/assignments/Parent.java]: You can return TYSON directly ,remove line no.5

File: src/main/java/com/training/assignments/Child.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.training.assignments;
+
+public class Child extends Parent {
+    public String getFirstName() {
+        String firstName = "MIKE";
+        return firstName;
```
Comment (shrikanttechincal) [inline src/main/java/com/training/assignments/Child.java]: return MIKE directly ,remove line 5

File: src/main/java/com/training/assignments/Test.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.training.assignments;
+
+public class Test {
+    public String getFullName() {
+        Child child = new Child();
+        child.getSurname();
+        child.getFirstName();
```
Comment (shrikanttechincal) [inline src/main/java/com/training/assignments/Test.java]: remove line 6,7

File: src/test/java/com/training/assignments/TestTest.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+
+class TestTest {
+    Test test = new Test();
+
+    @org.junit.jupiter.api.Test
+    public void testGetFullName() {
+        Assertions.assertEquals("TYSON MIKE", test.getFullName());
```
Comment (PranavHanwante) [inline src/test/java/com/training/assignments/TestTest.java]: new Test().getFullName()

Comment (shrikanttechincal) [inline src/test/java/com/training/assignments/TestTest.java]: Add assertion error  message

**Review by PranavHanwante — APPROVED** [review]

**Pull request merged**